test/export/StructureDefinition.LogicalExporter.test.ts

```diff
@@ -861,6 +861,48 @@ describe('LogicalExporter', () => {
     expect(loggerSpy.getAllMessages('error')).toHaveLength(0);
   });
 
+  it('should create Logical root element with short equal to title if short not available AND definition equal to description if definition not available', () => {
+    const logical = new Logical('MyTestModel');
+    logical.id = 'MyModel';
+    logical.title = 'MyTestModel title is here';
+    logical.description = 'MyTestModel description is here';
+
+    doc.logicals.set(logical.name, logical);
+    exporter.exportStructDef(logical);
+    const exported = pkg.logicals[0];
+
+    expect(exported.elements).toHaveLength(1);
+    expect(exported.elements[0].short).toBe(logical.title);
+    expect(exported.elements[0].definition).toBe(logical.description);
+  });
+
+  it('should create Logical root element with short equal to name if short and title not available AND definition equal to name if description and definition not available', () => {
+    const logical = new Logical('MyTestModel');
+    logical.id = 'MyModel';
+
+    doc.logicals.set(logical.name, logical);
+    exporter.exportStructDef(logical);
+    const exported = pkg.logicals[0];
+
+    expect(exported.elements).toHaveLength(1);
+    expect(exported.elements[0].short).toBe(logical.name);
+    expect(exported.elements[0].definition).toBe(logical.name);
+  });
+
+  it('should create Logical root element with short equal to title if short not available AND definition equal to short if description and definition not available', () => {
+    const logical = new Logical('MyTestModel');
+    logical.id = 'MyModel';
+    logical.title = 'MyTestModel title is here';
+
+    doc.logicals.set(logical.name, logical);
+    exporter.exportStructDef(logical);
+    const exported = pkg.logicals[0];
+
+    expect(exported.elements).toHaveLength(1);
+    expect(exported.elements[0].short).toBe(logical.title);
+    expect(exported.elements[0].definition).toBe(exported.elements[0].short);
+  });
+
```

Comment:
Are there any tests that ensure that if the user defined a rule to set the short and definition that the values from the rule take precedence? E.g., Given:
```
Logical: MyTestModel
Id: MyModel
Title: "MyTestModel title is here"
Description: "MyTestModel description is here"
* . ^short = "A short set by a rule"
* . ^definition = "A definition set by a rule"
```
Then the root element's short and definition should be the ones from the rules, not the title and description. If we already have a test for this, great! If we don't, can you add one?